website/docs/r/nsxt_edgegateway_dhcp_forwarding.html.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_edgegateway_dhcp_forwarding"
+sidebar_current: "docs-vcd-resource-nsxt-edge-dhcp-forwarding"
+description: |-
+  Provides a resource to manage NSX-T Edge Gateway DHCP forwarding configuration.
+---
+
+# vcd\_nsxt\_edgegateway\_dhcp\_forwarding
+
+Supported in provider *v3.10+* and VCD 10.3.1+ with NSX-T.
+
+Provides a resource to manage NSX-T Edge Gateway DHCP forwarding configuration.
+
+## Example Usage
+
+```hcl
+data "vcd_org_vdc" "v1" {
+  org  = "datacloud"
+  name = "nsxt-vdc-datacloud"
+}
+
+data "vcd_nsxt_edgegateway" "testing-in-vdc" {
+  org      = "datacloud"
+  owner_id = data.vcd_org_vdc.v1.id
+
+  name = "nsxt-gw-datacloud"
+}
+
+resource "vcd_nsxt_edgegateway_dhcp_forwarding" "testing-in-vdc" {
+  org             = "datacloud"
+  edge_gateway_id = data.vcd_nsxt_edgegateway.testing-in-vdc.id
+
+  enabled = true
+
+  dhcp_servers = [
+    "192.168.0.13",
+    "fe80::aaaa",
+  ]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) Org in which the NSX-T Edge Gateway is located, required
+  if not set in the provider section.
+* `edge_gateway_id` - (Required) NSX-T Edge Gateway ID.
+* `enabled` - (Required) DHCP Forwarding status. **If set to false, 
+  currently set DHCP servers won't be removed, which will result in an inconsistent plan** 
```

Comment:
I feel like understanding this note requires too much context knowledge of the user. First question that comes to mind after reading it is "_why_" it "will result in an inconsistent plan".

Would you elaborate and add more context? If it requires a lot of text, it could be done in a separate note, similar to here:
https://registry.terraform.io/providers/vmware/vcd/latest/docs/resources/nsxt_dynamic_security_group#attribute-reference